governance/GOVERNANCE.md

```diff
@@ -0,0 +1,116 @@
+
+### Subproject Owners
+
+Subproject owners are responsible for activities surrounding the development and release of content (eg. code, specifications, documentation) or the tasks needed to execute their subproject (e.g., community management) within the designated repository, or repositories associated with the subproject (e.g., community management). Technical decisions for code resides with the subproject owners unless there is a decision related to cross owners groups that cannot be resolved by those groups. Those cases can be escalated to the org owners.
+
+Subproject owners many be responsible for one or many repositories.
+
+Subproject owners do not need to be software developers. No explicit role is placed upon them and they can be anyone appropriate for the work being produced. For example, if a repository is for documentation it would be appropriate for owners to be technical writers.
+
+Changes to owners use the following:
+
+- A subproject owner may step down by submitting an [issue][oras-issues-new] stating their intent and they will be moved to emeritus.
+- Owners MUST remain active. If they are unresponsive for > 6 months they will be automatically removed unless a [super-majority][super-majority] of the other subproject owners agrees to extend the period to be greater than 6 months
+- New owners can be added to a subproject by a [super-majority][super-majority] vote of the existing owners
+- When a subproject has no owners the ORAS org owners become responsible for it and may archive the subproject or find new owners
+
+### Onboarding a New Owner
+
+New ORAS owners participate in an onboarding period during which they fulfill all code review and issue management responsibilities that are required for their role. The length of this onboarding period is variable, and is considered complete once both the existing owners and the candidate owners are comfortable with the candidate's competency in the responsibilities of ownership. This process MUST be completed prior to the candidate being named an official ORAS owner.
+
+The onboarding period is intended to ensure that the to-be-appointed owner is able/willing to take on the time requirements, familiar with ORAS core logic and concepts, understands the overall system architecture and interactions that comprise it, and is able to work well with both the existing owners and the community.
+
+## Decision Making at the ORAS org level
+
+When owners need to make decisions there are two ways decisions are made, unless described elsewhere.
+
+The default decision making process is [lazy-consensus][lazy-consensus]. This means that any decision is considered supported by the team making it as long as no one objects. Silence on any consensus decision is implicit agreement and equivalent to explicit agreement. Explicit agreement may be stated at will.
```

Comment:
Over how long a period? If I submit a proposal on Monday 0900 UTC, is it approved if no one objects by 0900 Tuesday? 0901 Monday (I hope not)? 0900 Monday a week later? We should have an explicit "time to object" if silence equals assent.

Reply:
Do we want acknowledgment of receipt, and do we need some quorum of acknowledgement for silence=assent to kick in (or voting)? How do we handle people being on vacation?

Reply:
I'm struggling on this one as it could be interpreted two ways. If we say we must have 3 business days for folks to object, does that mean that every PR either must have unanimous consent to approve that day, or every PR must wait 3 business days before merging because we must give time for the time clock to run out, or aggree?

Reply:
I think we need to distinguish between PRs - code changes - and major decisions. There are fundamental changes such as architecture or major new structures that should go through greater approval 9lazy consensus, give people time to object or agree), and there are the vast majority of PRs which really just takes one maintainer to sign off (fix a bug, add an option, add a new function, etc.).

If we distinguish between those, it becomes easier.

Reply:
I think this goes back to your questions about how much detail we need to provide, being overly prescriptive, to the point of legal mumbo-jumbo, and enough clarity for the right people to do the right things, the humans to make mistakes and let them be fixed, and enough to assure the few bad don't do bad things. (or allow bad things to be undone).

Reply:
> legal mumbo-jumbo

Yeah, I spend too much time in contracts; that's what I get for deciding to run my own business. 😆 

OK, let's have this "good enough" for now and improve later.